server/storage/rethinkdb_models.go

```diff
@@ -6,9 +6,10 @@ import (
 
 // These consts are the index names we've defined for RethinkDB
 const (
-	rdbSHA256Idx        = "sha256"
-	rdbGunRoleIdx       = "gun_role"
-	rdbGunRoleSHA256Idx = "gun_role_sha256"
+	rdbSha256Idx         = "sha256"
+	rdbGunRoleIdx        = "gun_role"
+	rdbGunRoleSha256Idx  = "gun_role_sha256"
```

Comment:
Group review:  SHA in a variable name should probably remain all caps, since it's an acronym as per go naming conventions.